### PR TITLE
Fixed make setup

### PR DESCRIPTION
**Pull request by mjvkeulen**
Minor tweaks to get the build script running again

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -94,7 +94,7 @@ setup-no-input:
 	(sudo apt-cache show python3.6 | grep "Package: python3.6") || \
 	(sudo add-apt-repository ppa:deadsnakes/ppa -y; sudo apt update) || echo "0"
 
-	sudo apt install nodejs -y
+	sudo apt install nodejs npm -y
```
Comment (engelhamer) [inline Makefile:97]: Npm should come with nodejs

Comment (mjvkeulen) [reply Makefile:97]: I had to add this to make the setup script work on a clean install

Comment (engelhamer) [reply Makefile:97]: Hm seems like a PPA may be the cause of differences here. Laat time I checked it was not needed. https://www.digitalocean.com/community/tutorials/how-to-install-node-js-on-ubuntu-18-04

Which version do you get from this install?

Comment (Larspolo) [reply Makefile:97]: Ubuntu 20.04 could be the difference here

Comment (mjvkeulen) [reply Makefile:97]: Setup for a fresh Ubuntu 20.04.2, npm version 7.13.0

Comment (Larspolo) [reply Makefile:97]: Problem is that we github acitons stil required dataclasses, I still manually comment is out because of that

Comment (mjvkeulen) [reply Makefile:97]: Cant we just move dataclasses to ci.txt?

File: requirements/ci.txt [hunk 2]
```diff
@@ -20,3 +20,4 @@ pytest-split==0.1.5 # Pytest with GitHub Actions speedup
 pytest-xdist==1.30.0 # Run Pytest in parallel
 deepdiff==5.0.2 # Testing deep difference of dictionary and list combinations
 django-test-migrations==0.3.0
+dataclasses==0.8 # Email sender config
```
Comment (Larspolo) [inline requirements/ci.txt:23]: Think this is Python version dependend, production still has an outdated version which should also be upgraded for it to not be needed there anymore

Comment (Larspolo) [reply requirements/ci.txt:23]: After #1150 is merged, this change is acceptable, but then it needs to be removed here as well

Comment (Larspolo) [reply requirements/ci.txt:23]: @mjvkeulen this should now be removed as well

**Review by Larspolo — CHANGES_REQUESTED** [review]

File: requirements/ci.txt (hunk 2 above)
Comment (engelhamer) [reply requirements/ci.txt]: It is removed

**Review by Larspolo — APPROVED** [review]
`nodejs` is no longer a valid command after this. I think its changed to just `node`

**Review by engelhamer — APPROVED** [review]

**Pull request merged**